src/sky-nav.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * This is a copy of the Sky Nav JavaScript with a few additions specifically
+ * for the demos.
+ *
+ * Create Sky Nav Menu
+ *
+ * Adds an event listener to the toggle button of a Sky Nav component for click
+ * events, which runs the toggle command to show or hide the menu with animation.
+ * Responds to breakpoint changes to show or hide the nav for large and small
+ * screens. Returns an object containing a `destroy()` method to remove the
+ * event listener.
+ *
+ * @param toggleEl - the toggle button for the navigation
+ */
+export const initSkyNav = (toggleEl: HTMLButtonElement) => {
+	let navButton = toggleEl;
+
+	// For the `:target` pseudo-class solution, the toggle is a link by default.
+	// Once this Sky Nav JS logic kicks in, we change the link to a button since
+	// using a button for this functionality is more semantic.
+	if (document.body.dataset.demo === 'target-pseudo-class') {
+		// Create a  new button
+		const button = document.createElement('button');
+		// Copy all of the link classes over to the new button
+		toggleEl.classList.forEach((toggleElCssClass) =>
+			button.classList.add(toggleElCssClass)
```

Comment:
It looks like `classList.add` accepts multiple, and `classList` is iterable so I think you can do:

```
something.classList.add(...otherEl.classList)`
```

Reply:
@calebeby I guess not? 🤷🏽 

<img width="931" alt="Screen Shot 2022-08-05 at 3 31 24 PM" src="https://user-images.githubusercontent.com/459757/183219742-8a40819c-6131-47db-aed9-24abe18cf197.png">